### DISCUSSION TITLE
Improving Rendering  on Windows

### DISCUSSION THREAD

**Discussion by Maikel1963** [RibirX/Ribir#608] [Q&A]
Hello
I found Ribir just a few weeks ago and it feels very promising. Following the Tutorial I have started to set up the TODOS example as own application.
But there is a problem with the rendering of the widgets. When compiling it for my private computer (Linux Endeavour with Vulkan graphics) it looks really nice and the letters have sharp edges.
Compiling it on my work computer (win10 ten years old with NVIDA Quattro) it looks bad. Not one sharp edge.
Does anybody know if I can something in the setup to improve the looks on the windows machine (besides getting a better computer)?

**Comment by M-Adoo** [comment]
Ribir's rendering quality is consistent across computers, as it only outputs triangles to the GPU. The observed differences may stem from monitor variations or potential bugs. Could you provide a screenshot to help us compare with other applications?

**Comment by Maikel1963** [comment]
![image](https://github.com/RibirX/Ribir/assets/17568498/8c698bdf-b095-4497-9fdd-e72ce2943a86)

**Reply by Maikel1963** [reply]
![image](https://github.com/RibirX/Ribir/assets/17568498/94f69ca8-d1ce-4472-9af5-feeda38ca4d5)

**Reply by Maikel1963** [reply]
And that's the one on the other computer (don't be irritated by the size - the word Todo seems to be of identical size on both computers, but the window size on start is different)

**Comment by Maikel1963** [comment]
Especially the list entries look clearer and sharper on my private computer

**Reply by M-Adoo** [reply]
Could you provide a screenshot of the text rendering from the older computer's browser for comparison?

What is the display resolution of this older computer?

Ribir is designed to be screen agnostic and does not utilize subpixel rendering. The subpixel rendering can improve the readability of small text on low-resolution screens. 

I haven't tested Ribir on a low-resolution display, so there could be unforeseen issues. I'm interested in comparing the quality to ensure it meets standards. My focus for June and July is on streamlining the core API framework. If there's an issue, I plan to address it subsequently. Achieving high performance and quality on older machines remains a key objective for Ribir.

**Reply by Maikel1963** [reply]
This is from the older computer (my private one is better :-)  When I am back home I'll make a screenshot from the newer one).
The resolution is 1920x1080 on this monitor. But the impression does not change when moving the TODOS-window to another monitor with lower resolution (1680x1050). There is no different pixelization. Even during the movement
That's why I guess the cause for  this unsharp depiction comes from wgpu.
